src/api/v1beta1/dynakube_types.go

```diff
@@ -104,7 +104,7 @@ type DynaKubeSpec struct {
 	// This property only affects certificates used to communicate with the Dynatrace API.
 	// The property is not applied to the ActiveGate
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Trusted CAs",order=6,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:io.kubernetes:ConfigMap"}
-	TrustedCAs string `json:"trustedCAs,omitempty"`
+	ClusterCa string `json:"trustedCAs,omitempty"`
```

Comment:
you can't just rename stuff in the CRD, there is a conversion webhook for alpha to beta, but this would cause a problem and would require a new conversion logic, which we don't want to do right now, and especially for the rename of a single filed

Reply:
json annotations are preserved. CRDs aren't affected.

Reply:
that is still not the most optimal way, as that is going to confuse us in a week or two after the change 😅